src/app/state/persist/index.ts

```diff
@@ -84,6 +85,12 @@ const persistSlice = createSlice({
     deleteProfileAsync(state) {
       state.loading = true
     },
+    resetWrongPassword(state) {
+      state.enteredWrongPassword = false
+    },
+    updatePassword(state, action: PayloadAction<UpdatePasswordPayload>) {
```

Comment:
I would move resetWrongPassword up to setWrongPassword
and rename updatePassword to updatePasswordAsync (convention in this file for saga-triggering actions)